### DISCUSSION TITLE
Empresas para se trabalhar

### DISCUSSION THREAD

**Discussion by camillodev** [frontendbr/forum#504] [Discussão]
Estou e procurando boas empresas para se trabalhar e crescer profissionalmente na area de Front End. 
Seria muito interessante ter opniões de pessoas que já trabalharam nelas, ou empresas que vocês querem trabalhar para eu pesquisar e colocar na minha lista. 
Pretendo analisa-las e aplicar para processos seletivos em breve,
Procuro principalmente empresas que se preocupamcom a experiência do usuario.
A minha lista até agora é:
- Lambda3
- Huge
- Globo.com
- Pagar.me
- Profite
- Grito
- OLX
- MundiPagg

**Comment by leonardosouzafaria** [comment]
MercadoLivre é uma empresa muito boa também!

**Comment by klaygomes** [comment]
boa sorte

**Comment by matheusml** [comment]
A [Concrete Solutions](http://www.concretesolutions.com.br/) está entre as [20 melhores empresas para se trabalhar](http://www.concretesolutions.com.br/2016/07/25/melhores-empresas-trabalhar/).

**Comment by camillodev** [comment]
@matheusml  você trabalha la? Conta um pouco sobre como é trabalhar ai e o time de desenvolvimento.

**Comment by ghost** [comment]
Vou apenas questionar um ponto @camillodev quando você fala em "boas empresas para se trabalhar e crescer" o que você avalia?

Te questiono isso por que existem empresas que tem um impacto significativo na comunidade, outras são mais "enterprise" ou ambientes mais fechados e até mesmo startups. são modelos bem diferentes.

Outro ponto interessante, o que importa pra você o desafio ou a qualidade de vida? Dependendo do seu momento ficar até tarde no trabalho é o mais importante ou que chegar cedo em casa pra ficar com seu filho.

Mas adicionaria também outras empresas como Nubank e VivaReal a sua lista. 

Pra falar da VivaReal invoco o @woliveiras que faz parte do VivaDecora ( me corrija se eu estiver errado rsrs).

**Comment by camillodev** [comment]
Cara, eu estou começando minha carreira ainda, então meu principal objetivo é crescimento profissional. Não me importaria nem um de ficar até tarde na empresa, trabalhar fds, portanto que seja pago por isso. Na minha atual fase, vale mais desafios do que qualidade de vida. 

Prefiro empresas mais envolvidas nas comunidades pois acredito que isso é mais uma forma de se manter atualizado e ganhar conhecimento., mas não que seria fator determinante. O mais importante é aprender, ter uma cultura da empresa bacana, e pagamentos justos. Pois tem muitas empresas que exigem absurdos e pagam muito pouco. Ou mesmo algumas que não pagam por "trabalhinhos extras". Essas empresas eu prefiro me manter fora. 

Então isso que eu queria ver mesmo. Comentários sobre como é o ambiente, se tem desafios, ou qualidade de vida. Se está sempre vinculada a comunidade. Se tem espaço para aprendizado e crescimento profissional.

**Comment by ghost** [comment]
@camillodev claro! Então agora acho que temos um parametro melhor pra avaliar se as empresas são boas ou não rsrs. Continue nessa livra, pessoas de outras regiões do Brasil talvez possam te indicar outros lugares legais também.

Te questionei isso por que as vezes as empresas pagam bem mas seu crescimento talvez não seja tão rápido quanto você imagina e vice-versa.

Mas continuaria nessa linha.

**Comment by bernardodiasc** [comment]
Aposte numa empresa que entende como funcionam as coisas no ramo, que lidem bem com o fato de que ninguém nasce sabendo e estudar e aprender FAZ PARTE do trabalho (e consome tempo/recurso mesmo).

**Comment by gabrieldarezzo** [comment]
Recomendo ouvir:
http://hipsters.tech/carreira-em-tecnologia-hipsters-31/

Sabias palavras do monstro chamado @akitaonrails

E dar uma lida nas referencias do repo abaixo:
https://github.com/woliveiras/front-end-career/blob/master/translations/pt-br/README.md

**Comment by camillodev** [comment]
@gabrieldarezzo  eu odeio podcast, sou mais visual, video ou leitura, mas ouvi esse podcast e cara é muito FODA, desculpem a palavra. 
Era tudo que eu precisava e tinha duvidas. Estou revendo todos meus planos de carreira. 
Muitissimo obbrigado mesmo.

**Comment by eduardojmatos** [comment]
@camillodev fica a dica: https://github.com/frontendbr/vagas/issues/292

**Comment by camillodev** [comment]
@eduardojmatos  obrigado pela recomendação.
Vou tentar sim, trabalhar no GetNinjas deve ser bem legal, principalmente pela descrição. 
Agora vamos ver se eles aceitam candidados do RJ.
Além disso eu sou pleno .Net mas especificamente Front ainda não tive a oportunidade. 
Mas não custa tentar. 

brigadão!

**Comment by eduardojmatos** [comment]
@camillodev saquei! Se estiver buscando um lugar bacana de trabalhar, recomendo :)

**Comment by fhott** [comment]
@camillodev sensacional a iniciativa!
Atualmente trabalho como front-end na Hotmart e é de longe a melhor empresa que trabalhei. Nunca cresci tanto em tão pouco tempo, além do ambiente ser excelente. O normal do dia-a-dia é desafio o tempo todo.

Tá aí o Love Mondays que não me deixa mentir: https://www.lovemondays.com.br/ranking/melhores-empresas-para-trabalhar-2017/pme

**Comment by caioincau** [comment]
O Moip tem se preocupado muito com UX/UI nesses últimos tempos, estamos mudando o modo como isso era visto aqui dentro.

**Comment by deppbrazil** [comment]
Tem as gigantes do [CPQD](https://www.cpqd.com.br/) o vale do silicio brasileiro, essas são empresas "great place to work" estão entre as 40 melhores empresas da América Latina para se trabalhar [CI&T](http://www.ciandt.com/home) e [Dextra](http://dextra.com.br/)

**Comment by woliveiras** [comment]
@edmolima foi mal pelo vácuo... Estava em um ritmo frenético de trampo e não conseguia acompanhar as issues com carinho. 

Eu trabalhava no Viva Decora até semana passada (14/06). 

Vou dizer um pouco do que passei lá:

**No VivaReal:**

- é uma das melhores empresas que eu conheço para se trabalhar.
- o salário é competitivo com o mercado e dependendo dos seus skills pode ser mais alto do que muitos lugares
- benefícios excelentes
- horário flexível
- ambiente descontraído, porém não bagunçado

**No VivaDecora:**

- o time é muito avançado e eu aprendi muito com eles em 1 ano e 7 meses de empresa, foi o lugar onde eu mais aprendi até hoje
- a gestão e diretoria é excelente em questão de ouvir os desenvolvedores - os devs tem poder de escolha e tomada de decisão final
- a cultura da empresa foi (e está sendo) construída pelos devs que estão fazendo parte do início dela
- é um time extremamente ágil (não pensando em Agile, mas na maneira como resolvem problemas)
- todo mundo se ajuda
- todo mundo pode dar palpite no código alheio
- todos focados em qualidade de software
- apesar de a stack front end parecer defasada pelo uso do Angular 1.x, Grunt e Sass (se você for do tipo que quer usar tudo o que está no DOM - Desenvolvimento Orientado a Modinha) a preocupação com arquitetura de software faz com que o desenvolvimento seja bem diferente do trabalho natural de algumas empresas que conheci (eu entrei no VD achando que sabia Angular)
- eles fazem reuniões individuais periodicamente para saber como você está se sentindo e o que você espera da empresa, se busca uma crescimento (quem não, né?), trocar de área ou coisa do tipo
- eles fazem uma avaliação geral semestral onde cada pessoa do time avalia o desempenho dos companheiros mais próximos o que pode gerar um feedback positivo, negativo ou até uma promoção (dependendo de seu cargo atual e várias outras variáveis, claro)
- campeonato de Mario Kart no Nintendo WII U nas reuniões mensais
- eles tem muito foco em SEO, então eu aprendi muito sobre isso lá também

É um ambiente extremamente favorável a quem curte discutir e trabalhar com Engenharia de Software.

No geral foi uma empresa que eu não tenho do que reclamar, mas se quiserem fazer alguma pergunta negativa podem me perguntar pra eu tentar falar mais, afinal eu posso estar enviesado por ter curtido muito trabalhar lá.

Quem quiser se candidatar para a minha vaga, esse aqui é o link: https://github.com/vivadecora/projeto-vaga-front-end-nao-fazer-fork

Não se esqueçam de dar uma estrelinha lá ou um watch para acompanhar, pois o pessoal está construindo o desafio nesse exato momento. 😬

**Comment by woliveiras** [comment]
Falando em como é trabalhar nas empresas, eu tenho um projeto em andamento que é exatamente sobre isso: Como é trabalhar na empresa X.

Quem quiser participar e escrever sobre o local de trabalho, pode entrar em contato comigo.

Aqui alguns exemplos: 

- https://medium.com/trainingcenter/como-é-trabalhar-na-entria-por-sibelius-seraphini-bb716427468c
- https://medium.com/trainingcenter/como-%C3%A9-trabalhar-no-grupo-caelum-por-s%C3%A9rgio-lopes-9923324971e

Seria show de bola se o @matheusml escrevesse sobre a Concrete, @eduardojmatos sobre a GetNinjas e o @icaioincau sobre a Moip, ein. 😄

**Comment by ghost** [comment]
@woliveiras sem problemas, legal saber que o Viva Real e o Viva Decora são lugares fantásticos. Eu realmente tinha esse insight de que eram lugares fantástico e entendo que existem ciclos e no teu caso talvez tenha sido isso.

Quanto ao projeto, se quiser posso falar da **Locaweb** =) E o que estamos produzindo após sofrer Rebrand. Temos bastante novidade no escopo de desenvolvimento usando cada vez mais linguagem como ferramenta e criando soluções mais robustas com arquiteturas e conceitos mais baixo nivél. Você deve ter visto a saída do Pothix ( William Molinari ) e o projeto de infradev dele. Enfim tem bastante coisa que podemos falar rsrs.


Obs: O projeto do trainingcenter sobre como é trabalhar na empresa X é além de fantástico para os devs quanto para as próprias empresas. Tenho um contato bem grande com o RH aqui da Locaweb e eles adoram as entrevistas pra ver perfis dos devs e o que eles curtem nas empresas no geral =)

**Comment by woliveiras** [comment]
\o/

Acho ótimo se fizer isso mano!

Tem muita gente que quer saber como é trabalhar na Locaweb, ein. 👍 

Me chama no Telegram pra gente trocar mais ideia sobre o post: `@woliveiras`

**Comment by lucasdaiki** [comment]
Um pouco da **Vizir** ❤️ , empresa DUCARALHO que trabalhei.
- Ambiente FODA. Quem conhece sabe o quão legal é trabalhar lá 😄. 
- Pessoas extremamente empenhadas em entregar produtos legais. Isso faz com que todo mundo evolua muito lá dentro.
- Projetos e clientes muito interessantes. Pude trabalhar com pessoas de várias partes do mundo.
- Horário flexível.
- Benefícios.

http://vizir.com.br/

**Comment by lucasdaiki** [comment]
Agora estou na **Easynvest**, o desafio aqui é bem legal. 
Estou há pouco tempo, só 3 meses mas já deu para sentir o clima por aqui :)

Para quem está na pegada de entregar uma experiência incrível para o usuário, aqui tem muita coisa para entregar. 
Tem um time bem legal de produto envolvido, os caras são realmente bons e dá para ver que todo o time tá na pegada de entregar algo foda. 
Time de front está com uma pegada muito da hora. Fazendo vários DOJOs e tudo mais. É tudo relativamente novo, tudo acontecendo muito rápido. Isso tá sendo a parte mais da hora. 

- Trampamos com React/Redux. 
- Ambiente ainda não é tão descontraído, mas está caminhando bem.
- Empresa crescendo muito. Bastante espaço para participar desse crescimento.
- Empresa incentiva MUITO (muito mesmo) o desenvolvimento do pessoal. Ajuda o pessoal a colar em eventos, ajuda em treinamentos... 
- Time tá na pegada!! 
- Mercado financeiro tinha uma mística de ser algo ultrapassado, velho e sem inovação. Aqui a empresa toda está lutando para mudar isso.  :)

**Comment by ghost** [comment]
@LucasDaiki eu ia pedir ao Wellyngton que deve trabalhar contigo pra falar sobre a Easynvest, mas aproveita e fala com o @woliveiras pra você falar um pouco de como é trabalhar lá =)

obs: Aliás ele é consultor, talvez não vale rsrsrs =)

**Comment by felipefialho** [comment]
Curti o tópico, tipo um LoveMondays 😹 

Vou seguir o ritmo e falar um pouco da área digital do [Cubo](https://cubo.network/) que é de longe o melhor lugar que trabalhei aqui no Brasil.

- Ambiente super agradável 
- Flexibilidade de horário e vestimenta
- Carta branca para escolher tecnologias
- Carta branca para escolher equipamentos
- Salário competitivo (ou até mesmo acima da média)
- Perspectiva de ter uma equipe altamente high-level
- Contato com algumas das melhores Startups do Brasil
- Liberdade para desenvolver projetos e soluções open-source
- Projetos inovadores que buscam de fato melhorar o empreendedorismo no Brasil
- Happy hours, comemorações e eventos quase diários organizados pelo próprio Cubo 

Temos [vagas abertas de Fullstack Developer](https://github.com/frontendbr/vagas/issues/354) para quem se interessar :)

**Comment by woliveiras** [comment]
@LFeh e @LucasDaiki topam falar dos trampos através de um post pro Training Center?

**Comment by felipefialho** [comment]
@woliveiras Topo ;)

**Comment by caioincau** [comment]
@woliveiras desculpa mas só vi a mention agora, se ainda estiver de pé o convite eu topo sim escrever :D (revisantando o tópico)

**Comment by woliveiras** [comment]
:raised_hands: 

Gente, do céu! Só lembrei hoje também

Vocês preferem que eu mande o doc pra vocês ou vocês mesmos escrevam nas suas contas no Medium (@LFeh e @caioincau)?

Ex,: 

- https://medium.com/trainingcenter/como-%C3%A9-trabalhar-na-entria-por-sibelius-seraphini-bb716427468c
- https://medium.com/trainingcenter/como-%C3%A9-trabalhar-no-grupo-caelum-por-s%C3%A9rgio-lopes-9923324971e

**Comment by felipefialho** [comment]
@woliveiras O doc seria com as perguntas pra seguir?

**Comment by caioincau** [comment]
Mesma dúvida do @LFeh  :D

**Comment by woliveiras** [comment]
Isso mesmo, galera :grin:

**Comment by felipefialho** [comment]
Pode enviar pra mim? Tento escrever essa semana ;)

**Comment by woliveiras** [comment]
Mandei para o seu email, @LFeh 

Pode me mandar seu email no meu Telegram (woliveiras), @caioincau?

**Comment by caioincau** [comment]
Enviado @woliveiras
